### ISSUE TITLE
NoMethodError: undefined method `current=' for Locale:Module

### ISSUE THREAD

**Issue by seven1m** [aurels/translation-gem#12]
I'm getting this error when trying to run `bundle exec rake --trace translation:init`:

```
** Invoke translation:init (first_time)
** Invoke environment (first_time)
** Execute environment
rake aborted!
NoMethodError: undefined method `current=' for Locale:Module
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/gettext-3.2.2/lib/gettext.rb:274:in `set_current_locale'
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/translation-1.9/lib/translation_io/railtie.rb:28:in `locale='
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/i18n-0.7.0/lib/i18n.rb:35:in `locale='
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/translation-1.9/lib/translation_io/railtie.rb:16:in `block in <class:Railtie>'
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-4.2.8/lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-4.2.8/lib/active_support/lazy_load_hooks.rb:45:in `block in run_load_hooks'
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-4.2.8/lib/active_support/lazy_load_hooks.rb:44:in `each'
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-4.2.8/lib/active_support/lazy_load_hooks.rb:44:in `run_load_hooks'
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/railties-4.2.8/lib/rails/application/finisher.rb:62:in `block in <module:Finisher>'
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/railties-4.2.8/lib/rails/initializable.rb:30:in `instance_exec'
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/railties-4.2.8/lib/rails/initializable.rb:30:in `run'
/usr/local/opt/rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/railties-4.2.8/lib/rails/initializable.rb:55:in `block in run_initializers'
...
```

I tried disabling gettext, but it had no effect.

Ideas?

Ruby: 2.3.3
Rails: 4.2.8

**Comment by aurels** [comment]
Hi,

Would you give us your `Gemfile.lock` and your translation initializer (without the API key of course).

You disabled gettext this way ? https://translation.io/blog/disable-gettext-in-your-project

**Comment by seven1m** [comment]
Here's the code: https://github.com/churchio/onebody/compare/master...translation-io

**Comment by MichaelHoste** [comment]
I found out the issue here. It's this file that was causing some trouble: https://github.com/churchio/onebody/blob/translation-io/lib/locale.rb

If you rename it (and rename the call here: https://github.com/churchio/onebody/blob/translation-io/config/initializers/_libs.rb) it will be fine. After the changes, I was able to successfully sync your translations with Translation.io.

FIY, Gettext was requiring the same name (we need gettext for internal stuff when initialising or syncing) and your file was loaded instead of the one of Locale gem.

PS : initialising your project may take some time, be patient. Once it's done, syncing will be faster.
